app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -13,9 +13,39 @@ def orders_volunteers
     @orders = Volunteer.find(params[:id]).orders
   end
 
+  def rating
+    @order_request = OrderRequest.find_by("order_id = ? AND order_request_status = ?", params[:order_id], OrderRequest.order_request_statuses[:accepted])
+    if @order_request != nil then 
+      @rating = VolunteerRating.new
+      @rating.order_id = params[:order_id]
+      @rating.qualifier_id = @order_request.volunteer.id
+      @rating.qualified_id = @order_request.order.helpee.id
+      @rating.score = params[:score]
+      @rating.comment = params[:comment]
+      @rating.save!
```

Comment:
```suggestion
      rating = VolunteerRating.new
      rating.order_id = params[:order_id]
      rating.qualifier_id = @order_request.volunteer.id
      rating.qualified_id = @order_request.order.helpee.id
      rating.score = params[:score]
      rating.comment = params[:comment]
      rating.save!
```
Lo mismo que en el otro controlador.